parm/atm/obs/config/amsua_n19.yaml

```diff
@@ -20,7 +20,7 @@ obs operator:
   obs options:
     Sensor_ID: amsua_n19
     EndianType: little_endian
-    CoefficientPath: !ENV ${DATA}/crtm
+    CoefficientPath: !ENV ${DATA}/crtm/
```

Comment:
Yes I noticed this is an issue, we'll need to fix the other YAMLs in a new PR soon I think

Reply:
I forgot about this change.   I agree.  We can address this in a new PR.